go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ubuntu/gowsl
 
-go 1.21
+go 1.22.1
```

Comment:
any reason to pin `1.22.1` instead of `1.22.0`? (then adding toolchain as needed).